.github/CODEOWNERS

```diff
@@ -0,0 +1,8 @@
+# Users referenced in this file will automatically be requested as reviewers for PRs that modify the given paths.
+# See https://help.github.com/articles/about-code-owners/
+
+*                                                                                 @dotnet/aspnet-blazor-eng
+/.github/                                                                         @dotnet/aspnet-build @dougbu @wtgodbe
+/eng/                                                                             @dotnet/aspnet-build @dougbu @wtgodbe
+/eng/common/                                                                      @dotnet-maestro-bot
+/eng/Versions.props                                                               @dotnet-maestro-bot @dotnet/aspnet-build @dougbu @wtgodbe
```

Comment:
https://github.com/dotnet/aspnetcore/blob/main/.github/CODEOWNERS#L16

Errors consistent with aspnetcore. cc/ @dotnet/aspnet-build

Reply:
Remove the `dotnet-maestro-bot` user from these lines. It adds no value because we have never used the merge "bot" in this repo.